spec/stack.js

```diff
@@ -11,12 +11,48 @@ describe('Stack', () => {
     expect(Stack).to.be.a('function')
   })
 
+  context('length()', () => {
+    it('returns the size of the stack', () => {
+      const myStack = new Stack()
+      myStack.push('foo')
+      myStack.push('bar')
+      expect(myStack.length()).to.eql(2)
+    })
+  })
+
+  context('isEmpty()', () => {
+    it('returns true if the stack is empty', () => {
+      const myStack = new Stack()
+      myStack.push('foo')
+      myStack.push('bar')
+      expect(myStack.isEmpty()).to.eql(false)
+    })
+  })
+
   context('push()', () => {
     it('pushes an element to the top of the stack.', () => {
       const myStack = new Stack()
+      myStack.push('foo')
```

Comment:
You would also want to test that the element pushed was the correct element - this only tests that the stack length changed when pushing.  

Consider implementing `Stack` with a linked list implementation, instead of re-using the native javascript implementation, to ensure familiarity with the behaviors.